zerocopy-derive/tests/ui-nightly/absence_of_deprecated_warning.rs

```diff
@@ -0,0 +1,24 @@
+//! See: https://github.com/google/zerocopy/issues/553
```

Comment:
```suggestion
// Copyright 2024 The Fuchsia Authors
//
// Licensed under a BSD-style license <LICENSE-BSD>, Apache License, Version 2.0
// <LICENSE-APACHE or https://www.apache.org/licenses/LICENSE-2.0>, or the MIT
// license <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your option.
// This file may not be copied, modified, or distributed except according to
// those terms.

//! See: https://github.com/google/zerocopy/issues/553
```

Reply:
Ah yes, I was wondering about that. The contributing guide didn't mention it but probably should.